PlayerUI/ViewStrategy/objc/Podfile

```diff
@@ -0,0 +1,8 @@
+source 'https://github.com/brightcove/BrightcoveSpecs.git'
+
+platform :ios, '11.0'
+use_frameworks!
+
+target 'ViewStrategy' do
+	 pod 'Brightcove-Player-Core'
```

Comment:
Because of the player layer issue, should this be `pod 'Brightcove-Player-Core', '>= 6.7.8'`?

Reply:
Yes, @timothyrodgers